cmd/soci/commands/index/info.go

```diff
@@ -80,6 +79,6 @@ func prettyPrintJSON(b []byte) error {
 	if err := json.Indent(&prettyJSON, b, "", "  "); err != nil {
 		return err
 	}
-	_, err := os.Stdout.Write(prettyJSON.Bytes())
-	return err
+	fmt.Println(prettyJSON.String())
+	return nil
```

Comment:
nit: shouldn't this return the error from `fmt.Println` if there is one?

Reply:
TIL: print line returns an error.